### DISCUSSION TITLE
vm abnormal shutdown while cbt backup working

### DISCUSSION THREAD

**Discussion by daxiguafan** [abbbi/virtnbdbackup#105] [Q&A / Support] [answered]
when vm（or host） abnormal shutdown happen while cbt backup working. does it need full backup again? how to optimize the performance？

**Comment by abbbi** [comment, marked as answer]
yes, it does need full backup again, usually if virtnbdbackup detects an error it should error out accordingly and leave an .partial file within the target backup folder. if it finds this partial file, then no further incremental backups will work  and a new full backup is required.

What exactly do you mean with optimizing the performance? Backup performance depends on various other components such as load of the host system, used compression, target backup storage speed, etc.

**Reply by daxiguafan** [reply]
Can scan and compare the disk to aviod full backup again? 
refer vmware‘s  Initial full sync，use replication “seeds”？
https://blogs.vmware.com/vsphere/2015/06/vsphere-replication-synchronization-types.html

**Reply by abbbi** [reply]
nothing like this is implemented at current.
